force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -163,126 +196,164 @@ public with sharing class ClassesObjectsMethods {
-     * where you need to manipulate and manage book information.
-     */
+    /*
+    //     // Subtract the difference in years
+
+    //      // Replace null with the variable you used to store the result
+    // }
+
+    // /**
+    //  * Question 11
+    //  * The Book class represents a book in a library. 
+    //  * Each Book object has a title and an author, and methods to return these details.
+    //  * This class can be used in the context of a library management system, bookstore, etc.
+    //  * where you need to manipulate and manage book information.
+    //  */
     public class Book {
         // Declare three public instance variables - title, author, pages
         // title
         // author
         // pages
-    
+       public String title;
+       public String author;
+       public String pages;
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(String title, String author) {
+            this.title = title;
+            this.author = author;
+
             // Initialize the title and author
         }
     
         // Method that returns the details of the book
         public String getBookDetails() {
             // return A string containing the details of the book in the format "Title: <title>, Author: <author>"
-            return null; // Replace null with the variable you used to store the result
+            String bookdetails = 'Title: ' + this.title + ', Author: ' + this.author;
+             
+            return bookdetails; // Replace null with the variable you used to store the result
         }
     }
     
 
-    /**
-     * Question 12
-     * Create a Book object and returns the details of the book.
-     * For example, createAndGetBookDetails() should return a string like "Title: A Brief History of Time, Author: Stephen Hawking".
-     * @return A string containing the details of the book.
-     */
+    // /**
+    //  * Question 12
+    //  * Create a Book object and returns the details of the book.
+    //  * For example, createAndGetBookDetails() should return a string like "Title: A Brief History of Time, Author: Stephen Hawking".
+    //  * @return A string containing the details of the book.
+    //  */
     public static String createAndGetBookDetails() {
-        // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
-
-        // Call the getBookDetails method on the book object to get the details of the book
+    //     // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
+    //     Book_c = book = [Select Id, Title_c, Author_C, Publishe_c, Isbn_c, From Book];
+        Book newBook = new Book('A Brief History of Time', 'Stephen Hawking');
+      
+    //     // Call the getBookDetails method on the book object to get the details of the book
+
+     
+  
+    return newBook.getBookDetails();
```

Comment:
Good. I would clean up all the extra while space in your code. It makes it hard to read. 

You could do this with out any temp variables. If you want. 

   public static String createAndGetBookDetails() {
        return new Book('A Brief History of Time', 'Stephen Hawking').getBookDetails();
    }